arangod/Agency/Node.cpp

```diff
@@ -581,8 +582,13 @@ bool Node::handle<REPLACE>(VPackSlice const& slice) {
   Builder tmp;
   { VPackArrayBuilder t(&tmp);
     if (this->slice().isArray()) {
+      VPackSlice valToRepl = slice.get("val");
       for (auto const& old : VPackArrayIterator(this->slice())) {
-        tmp.add(old == slice.get("val") ? slice.get("new") : old);
+        if (VelocyPackHelper::compare(old, valToRepl, /*useUTF8*/true) == 0) {
```

Comment:
We are checking for equality only here, not for < or >. Can't we set `useUtf8` to false then?

Reply:
I don't know technically the docs say just equality, so unicode equality would have to be accounted for ?